package.json

```diff
@@ -11,30 +11,32 @@
     "react-component"
   ],
   "dependencies": {
-    "dehumanize-date": "^2.0.1",
-    "occasion": "^1.0.0",
-    "prop-types": "^15.5.8"
+    "dehumanize-date": "2.4.0",
+    "occasion": "2.0.0",
+    "prop-types": "^15.5.10",
+    "react-dom": "^15.6.1",
+    "react-test-renderer": "^15.6.1"
```

Comment:
react-dom and react-test-render need to be devDependencies, not dependencies (and react needs to remain a peer-dependency & devDependency.